src/colorizer/Dataset.ts

```diff
@@ -111,6 +111,7 @@ export default class Dataset {
 
   private resolveUrl = (url: string): string => `${this.baseUrl}/${url}`;
 
+  // TODO: Make this part of `urlUtils`?
```

Comment:
Makes sense to me - I'm strongly in favor of moving class methods that never use `this` out of their class.